src/worker/index.ts

```diff
@@ -70,13 +70,30 @@ export class IFrameWorker implements Worker {
     /* Create iframe to host the worker script */
     const iframe = document.createElement("iframe")
     iframe.width = iframe.height = iframe.frameBorder = "0"
-    document.body.appendChild(this.iframe = iframe)
+    
+    /* Tells browsers to treat iframe as if from another origin */
+    iframe.setAttribute('sandbox','allow-scripts');
+    
+    
 
     /* Initialize runtime and worker script */
-    this.w.document.open()
-    this.w.document.write(
+    /* Encode the iframe page into a data URI */
+    const dataUri = "data:text/html;base64," + 
+    btoa(
       "<html>" +
         "<body>" +
+      /* Attempt to change the origin of the iframe using a parent domain suffix */
+          "<script>" +
```

Comment:
This code will not get minified appropriately, increase the size of the bundle. Could you please move this into a separate function, so ESBuild can do its job? See `postMessage` and `importScripts` below.

Reply:
I'm thinking to remove this section. It addresses edge cases outside of your support.